google/cloud/spanner_v1/instance.py

```diff
@@ -306,13 +315,14 @@ def update(self):
 
         .. note::
 
-            Updates the ``display_name``, ``node_count`` and ``labels``. To change those
-            values before updating, set them via
+            Updates the ``display_name``, ``node_count``, ``processing_units``
+            and ``labels``. To change those values before updating, set them via
 
             .. code:: python
 
                 instance.display_name = 'New display name'
                 instance.node_count = 5
+                instance.processing_unit = 2000
```

Comment:
This is invalid. You can't set both `node_count` and `processing_units`. Either revert this or remove the `node_count` value.